pytket/binders/circuit/main.cpp

```diff
@@ -74,6 +74,7 @@ PYBIND11_MODULE(circuit, m) {
           py::arg("latex") = false)
       .def("__eq__", &Op::operator==)
       .def("__repr__", [](const Op &op) { return op.get_name(); })
+      .def("free_symbols", [](const Op *op) { return op->free_symbols(); })
```

Comment:
Can we pass by const reference rather than address, unless there's some reason why this wouldn't work?

Reply:
No reason. Done. Out of curiosity, or maybe just a naive question. Why is a reference preferred ? Because it offers more security guarantees than a pointer ?

Reply:
In this case it probably makes very little difference, but in general, passing pointers around is less safe than passing references (in the sense that bugs leading to segfaults or UB are more likely), so for preference we should pass by reference.

Reply:
Thanks. Yeah that's what I've been reading around. Mostly the unsafeness around the nullability of pointers.